Makefile

```diff
@@ -329,6 +329,7 @@ test: gotestsum
     -e "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/auth/helper.go" \
     -e "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/auth/context_config.go" \
     -e "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/client/keycloak/config.go" \
+	-e "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/services/sentry/providers.go" \
```

Comment:
I think we should ignore `pkg/services/sentry/config.go` as well (keep the test though). Also please add one more test for `ReadFiles()` in this file, you can find example here: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/bd75b56ea197b53e97fe1abbacad029c9f1ca361/internal/kafka/internal/config/aws_test.go#L35

Reply:
Thanks @pawelpaszki